docs/schema/guidance/repr-beneficial-ownership.rst

```diff
@@ -24,52 +26,58 @@ In BODS, the following properties are used to represent such information disclos
+1. There MUST be a *primary* relationship connecting the two which represents the beneficial ownership relationship. Specifically, in the ``recordDetails`` for the relationship: 
 
-   a. the entity’s Statement MUST be the ``subject``;
-   b. the person’s Statement MUST be the ``interestedParty``;
+   a. the entity’s record MUST be the ``subject``;
+   b. the person’s record MUST be the ``interestedParty``;
    c. ``isComponent`` MUST be false;
-   d. the ``interests`` which make the person meet the criteria for their being declared a beneficial owner MUST be included in this primary Ownership-or-control Statement if known; and
-   e. the ``interests`` in (d) MUST have ``beneficialOwnershipOrControl`` set to 'true'.
-
-2. If beneficial ownership is known to be exercised indirectly, via intermediary entities then ``directOrIndirect`` MUST be ‘indirect’. If it is known to be exercised directly then ``directOrIndirect`` MUST be ‘direct’. Otherwise ``directOrIndirect`` MUST be ‘unknown’.
+   d. the ``interests`` which make the person meet the criteria for their being declared a beneficial owner MUST be included if known; and
+   e. the ``interests`` in (d) MUST have ``beneficialOwnershipOrControl`` set to 'true'. If an interest is known to be exercised indirectly, via intermediary entities then ``directOrIndirect`` MUST be ‘indirect’. If it is known to be exercised directly then ``directOrIndirect`` MUST be ‘direct’. Otherwise ``directOrIndirect`` MUST be ‘unknown’.
 
 Intermediaries
 ^^^^^^^^^^^^^^
 
-Where beneficial ownership is known to be exercised indirectly, via known intermediary entities, this SHOULD be represented in addition to the above. In particular:
+Where beneficial ownership is known to be exercised indirectly, via known intermediary elements, this SHOULD be represented in addition to the above. In particular:
 
-1. The chain of known intermediary entities SHOULD be represented by *secondary* Entity Statements, Ownership-or-control Statements and Person Statements.
-2. These secondary statements SHOULD link the beneficial owner’s Statement to entity X’s statement indirectly.
-3. These secondary statements SHOULD all have ``isComponent`` set to 'true'.
-4. These secondary statements SHOULD all have their ``statementID`` values listed in the ``componentStatementIDs`` array of the primary Ownership-or-control Statement.
-5. When the primary Ownership-or-control Statement is published in a BODS file:
+1. The chain of known intermediary elements SHOULD be represented by *secondary* records.
+2. These secondary records SHOULD link the beneficial owner’s record to entity X’s record indirectly.
+3. These secondary records SHOULD all have ``isComponent`` set to 'true'.
+4. These secondary records SHOULD all have their ``recordId`` values listed in the ``componentRecords`` array of the ``recordDetails`` for the primary relationship.
+5. When ``recordDetails`` for the primary relationship are published in a BODS file:
```

Comment:
```suggestion
When ``recordDetails`` for the primary relationship are published in a BODS file:
```

I think this should be a separate point as it is about the publishing not the data itself? Also lists within lists gets a bit complicated 

Also I find the reference to recordDetails a bit confusing here - I think this should be relationship Record / Statement

Reply:
Yup.